config/routes.rb

```diff
@@ -1,6 +1,3 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
-  # Defines the root path route ("/")
-  # root "articles#index"
+  resources :users, :posts, :pages
```

Comment:
- [ ] please avoid unnecessarily exposing end-points, your routes might look something like this to avoid exposing them.
```
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end

```

Reply:
Thank you @mikethreels for the feedback and correction